tests/general-api-tests/axis.spec.ts

```
@@ -0,0 +1,384 @@
+      test.step(`Check quotient (${actualQuotient}) steps values`, async () => {
+        const zeroValuesNumber = axis.steps.reduce((acc: number, step) => {
+          const value = step.value;
+          expect
+            .soft(
+              value,
+              `Step value (${value}) should be defined for ${actualQuotient}`
+            )
+            .toBeDefined();
+          expect
+            .soft(
+              isFinite(value),
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)